.github/workflows/maven.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         java-version: 11.0.16+8
         distribution: 'temurin'
     - name: Cache Maven packages
-      uses: actions/cache@v2
+      uses: actions/cache@v4
```

Comment:
Is this change intentional?

Reply:
Yes, the checks of all pending PRs are failing due to using this deprecated v2 version.